WordPress/Classes/Extensions/String+Helpers.swift

```diff
@@ -28,6 +28,12 @@ extension String {
         return copy
     }
 
+    func stripHtmlTags() -> String {
```

Comment:
I like that this uses Apple's preferred method of using `NSAttributedString` with `NSHTMLTextDocumentType`, but there are some potential snags with performance or lockups that make me a bit wary of making this change right now.

We happen to have `stringByStrippingHTML` in the codebase already. Can you rework your changes to use this instead?